src/frontend/components/PublicVideoDashboard/index.spec.tsx

```diff
@@ -0,0 +1,281 @@
+                { display_name: 'English', value: 'en' },
+                { display_name: 'French', value: 'fr' },
+              ],
+            },
+          },
+        },
+      },
+      { method: 'OPTIONS' },
+    );
+    mockCreatePlayer.mockResolvedValue({
+      destroy: jest.fn(),
+    });
+  });
+
+  afterEach(() => {
+    fetchMock.restore();
+    jest.clearAllMocks();
+  });
+  it('displays the video player alone', async () => {
+    useTimedTextTrackStub.returns([]);
+    const video = videoMockFactory({
+      urls: {
+        manifests: {
+          hls: 'https://example.com/hls.m3u8',
+        },
+        mp4: {
+          144: 'https://example.com/144p.mp4',
+          1080: 'https://example.com/1080p.mp4',
+        },
+        thumbnails: {
+          144: 'https://example.com/thumbnail/144p.jpg',
+          1080: 'https://example.com/thumbnail/1080p.jpg',
+        },
+      },
+    });
+
+    const { container } = render(
+      wrapInIntlProvider(
+        <PublicVideoDashboard video={video} playerType="videojs" />,
+      ),
+    );
+
+    await waitFor(() =>
+      // The player is created and initialized with DashJS for adaptive bitrate
+      expect(mockCreatePlayer).toHaveBeenCalledWith(
+        'videojs',
+        expect.any(Element),
+        expect.anything(),
+        video,
+      ),
+    );
+
+    expect(
+      container.querySelector('source[src="https://example.com/144p.mp4"]'),
+    ).not.toBeNull();
+    expect(
+      container.querySelector('source[src="https://example.com/1080p.mp4"]'),
+    ).not.toBeNull();
+    expect(container.querySelectorAll('source[type="video/mp4"]')).toHaveLength(
+      2,
+    );
+    const videoElement = container.querySelector('video')!;
+    expect(videoElement.tabIndex).toEqual(-1);
+    expect(videoElement.poster).toEqual(
+      'https://example.com/thumbnail/1080p.jpg',
+    );
+  });
+  it('displays the video player, the download link and transcripts', async () => {
+    const timedTextTracks = [
+      timedTextMockFactory({
+        is_ready_to_show: true,
+        mode: timedTextMode.TRANSCRIPT,
+      }),
+    ];
+    useTimedTextTrackStub.returns(timedTextTracks);
+    const video = videoMockFactory({
+      has_transcript: true,
+      show_download: true,
+      timed_text_tracks: timedTextTracks,
+      urls: {
+        manifests: {
+          hls: 'https://example.com/hls.m3u8',
+        },
+        mp4: {
+          144: 'https://example.com/144p.mp4',
+          1080: 'https://example.com/1080p.mp4',
+        },
+        thumbnails: {
+          144: 'https://example.com/thumbnail/144p.jpg',
+          1080: 'https://example.com/thumbnail/1080p.jpg',
+        },
+      },
+    });
+
+    const { container } = render(
+      wrapInIntlProvider(
+        <PublicVideoDashboard video={video} playerType="videojs" />,
+      ),
+    );
+
+    await waitFor(() =>
+      // The player is created and initialized with DashJS for adaptive bitrate
```

Comment:
Maybe I'm wrong here, but it seems that [dashjs has been removed](https://github.com/openfun/marsha/commit/6a777ab9c7b6448a94b4c4bf454be226a93d4cdd)

Reply:
You are right